CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Add support for Mac Catalyst to App Center Analytics and App Center Crashes. Als
 * **[Feature]** Add support for Mac Catalyst.
 * **[Fix]** Fix `Undefined symbol: ___llvm_profile_runtime` when the SDK was built from source by Carthage.
 * **[Fix]** Fix screen resolution detection macOS.
+* **[Fix]** Fix crash when NSData is corrupted.
```

Comment:
What data? Please specify what data it is from customer's view. Class name of data structure says nothing